### PR TITLE
Drinking soy now makes you gay, also gay trait.

### PR DESCRIPTION
**Pull request by ghost**
:cl: Floyd
add: Homosexuality trait. you dont wanna hug the opposite sex
add: Soy now makes you gay
/:cl:

my memorial to our man Aniki

oranges> we need a trait homosexual
oranges> that makes you not enjoy hugs from woman
oranges> and only from men
oranges> and then we can add a gas that makes you gay

image

they put chemicals in the water etc

### CODE REVIEW

**Comment by Qustinnus** [comment]
@optimumtact happy now

**Comment by Armhulen** [comment]
record timing confused emoji on your pr

**Comment by Qustinnus** [comment]
they wouldnt understand me

**Comment by Screemonster** [comment]
are the globalists putting chemicals in the water?

**Comment by Armhulen** [comment]
wtf duide....................................

**Comment by Qustinnus** [comment]
et tu floyd?

**Comment by TechnoAlchemisto** [comment]
this sucks yo

**Comment by Pacmandevil** [comment]
soyboy detected

**Comment by davethwave** [comment]
so people turn gay if they drink more than 15u of soy?

**Comment by Armhulen** [comment]
now we need the vent clog event but it only dispenses soy

**Comment by Iamgoofball** [comment]
Can we not add subtle racist memes to the codebase

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 1]
```diff
@@ -263,7 +263,14 @@
 						"<span class='notice'>You hug [src] to make [p_them()] feel better!</span>")
 			GET_COMPONENT_FROM(mood, /datum/component/mood, src)
 			if(mood)
-				mood.add_event("hug", /datum/mood_event/hug)
+				var/mob/living/L = src
+				if(L.has_trait(TRAIT_HOMOSEXUAL))
+					if(M.gender == L.gender)
+						mood.add_event("hug", /datum/mood_event/gayhug)
+					else
+						mood.add_event("hug", /datum/mood_event/badhug) //Gays are picky I guess?
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/carbon_defense.dm]: Kinda offensive my dude

**Review by Iamgoofball — CHANGES_REQUESTED** [review]

**Comment by ShadowDeath6** [comment]
should make it so that the jabroni outfit has a chance of making you gay tbh

**Comment by Repukan** [comment]
Wearing the Jabroni outfit should give you a good moodlet if gay

**Comment by Iamgoofball** [comment]
^ yeah those would be better

**Comment by kevinz000** [comment]
what if i like soymilk

**Comment by KorPhaeron** [comment]
Do we really need a bunch of 15 year olds running around shouting soyboy at one another?

**Comment by GuyonBroadway** [comment]
That sounds like your typical autistic spaceman anyway.

**Comment by duncathan** [comment]
this is just garbage

**Comment by Grazyn** [comment]
\>use gay as an insult
\>"b-but gay isn't an insult! Get off my back fucking SJWs!"
\>hey guys what if we make up a word that means gay and use that as an insult
\>so cunningly devilish!

the history of soyboy

**Comment by GuyonBroadway** [comment]
I thought soyboys were just beta male hipster cucks.

**Comment by MoreRobustThanYou** [comment]
oh my god

File: code/datums/traits/neutral.dm [hunk 2]
```diff
@@ -10,7 +10,13 @@
...
 	medical_record_text = "Patient suffers from ageusia and is incapable of tasting food or reagents."
 
-
+/datum/trait/gay
+	name = "Homosexual"
+	desc = "You prefer people of your own gender."
+	value = 0
+	mob_trait = TRAIT_HOMOSEXUAL
+	gain_text = "<span class='notice'>You suddenly feel like you prefer your own gender.</span>"
+	lose_text = "<span class='notice'>You feel like you prefer other genders again.</span>"
```
Comment (Irafas) [inline code/datums/traits/neutral.dm]: If this is a trait that can selected in character preferences this line doesn't really make sense as it implies that the character was originally straight.  Should probably be 
`lose_text = "<span class='notice'>You feel like you prefer other genders.</span>" `

**Comment by Cyberboss** [comment]
All greytiders will do for a while now is stuncuff and forcefeed soy

**Comment by Militaires** [comment]
#24407

**Comment by ExcessiveUseOfCobblestone** [comment]
projecting through prs are we floyd?

**Comment by Spaceinaba** [comment]
gay trait is good and funny
jabroni outfit should make you gay
soy is the only bad part of this pr

File: code/modules/mob/living/carbon/carbon_defense.dm (hunk 1 above)
Comment (tortellinitony) [reply code/modules/mob/living/carbon/carbon_defense.dm]: What if they are a neuter? Does that count?

**Comment by Qustinnus** [comment]
>wake up
>more than 3 serious replies to this


boi

File: code/modules/mob/living/carbon/carbon_defense.dm (hunk 1 above)
Comment (Techhead0) [reply code/modules/mob/living/carbon/carbon_defense.dm]: Everyone should enjoy hugs. Just give a stronger positive moodlet on a preferred-gender hug and a weaker moodlet on other hugs.

Comment (Qustinnus) [reply code/modules/mob/living/carbon/carbon_defense.dm]: @Iamgoofball you know damn well I'm the embodiment of everything soy and gay in the modern world and im not offended :^)

**Comment by Qustinnus** [comment]
fruit pls

**Comment by Qustinnus** [comment]
i do not want to be killed by the github admins

**Comment by epicmemename** [comment]
first moodlets now this, hmmmm makes you think

**Comment by PKPenguin321** [comment]
if it was jabroni outfit instead of soy this pr would be A+

**Comment by optimumtact** [comment]
I suppose we are violating their ToS right now with our alt right hate speech

**Comment by Vivalas** [comment]
I remember once when a Github staff member casually tried to implement a "Terms of Contributing" to a codebase mostly centered around a resource for teaching people how to code.. mostly because someone else made a PR that tried to remove a "coding chicks" link because it was "offensive to women" and people called out the bullshit...


Ahhhh.... Silicon Valley...

**Pull request closed without merging**